src/main/api/ledger.ts

```diff
@@ -1,14 +1,20 @@
 import AppBtc from '@ledgerhq/hw-app-btc'
 import TransportNodeHid from '@ledgerhq/hw-transport-node-hid'
+import { Network } from '@xchainjs/xchain-client'
 import * as E from 'fp-ts/Either'
 
 import { LedgerErrorId } from '../../shared/api/types'
 
-export const getBTCAddress = async () => {
+export const getBTCAddress = async (network: Network) => {
   try {
     const transport = await TransportNodeHid.open('')
     const appBtc = new AppBtc(transport)
-    const info = await appBtc.getWalletPublicKey("44'/0'/0'/0/0")
+    let info
+    if (network === 'mainnet') {
+      info = await appBtc.getWalletPublicKey("44'/0'/0'/0/0")
```

Comment:
just a nitpick
could you pls move ` "44'/0'/0'/0/0"` and etc to the semantically named constants and description about WHY they have such values ?